agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -85,7 +85,7 @@
 
     public static class Sampling {
 
-        public double percentage = 100;
+        public float percentage = 100;
```

Comment:
We use double everywhere.. wondering why choose to use float here?

Reply:
the new TelemetryItem.setSampleRate takes a float, so that change rippled to several places